modules/ocf_kubernetes/functions/gen_aliases.pp

```diff
@@ -0,0 +1,29 @@
+# Expands ['mastodon', ('pma', ['phpmyadmin'])]
+# to { 'mastodon.ocf.berkeley.edu' => 'mastodon.ocf.io', 'mastodon']
+#      'pma.ocf.berkeley.edu       => ['pma.ocf.io', 'pma', 'phpmyadmin.ocf.berkeley.edu',
+#                                      'phpmyadmin.ocf.io', 'phymyadmin'] }
+
+function ocf_kubernetes::gen_aliases(Array $services) >> Hash[String, Array[String]] {
```

Comment:
I'm not a huge fan of the amount of "magic" that I feel this function is doing, but big thanks for including types for it and I think it's still useful to have. I'd say to keep it, just a bit reluctantly :P

Reply:
I found a solution that doesn't involve this file.